IBPSA/ThermalZones/ISO13790/Zone5R1C/Zone.mo

```diff
@@ -0,0 +1,327 @@
+    annotation (Placement(transformation(extent={{0,-10},{20,10}})));
+  Modelica.Thermal.HeatTransfer.Components.ThermalConductor HThe(G=3.45*AFlo*4.5) "Coupling conductancec betwee air and surface nodes"
+    annotation (Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=90,
+        origin={40,40})));
+  Modelica.Thermal.HeatTransfer.Components.ThermalConductor HMas(G=9.1*facMas*AFlo)
+    "Coupling conductancec between surface and mass nodes" annotation (
+      Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=90,
+        origin={40,-40})));
+  Modelica.Thermal.HeatTransfer.Components.HeatCapacitor capMas(C=buiMas.heaC*
+        AFlo, T(
+      displayUnit="K",
+      fixed=true,
+      start=293.15)) "Zone thermal capacity" annotation (Placement(
+        transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=180,
+        origin={40,-120})));
+  Modelica.Thermal.HeatTransfer.Interfaces.HeatPort_a TAir "Air node"
+    annotation (Placement(transformation(extent={{30,70},{50,90}})));
+  Modelica.Thermal.HeatTransfer.Interfaces.HeatPort_a TSur "Surface node"
+    annotation (Placement(transformation(extent={{30,-10},{50,10}})));
+  Modelica.Thermal.HeatTransfer.Sources.PrescribedHeatFlow heaAir
+    annotation (Placement(transformation(extent={{80,70},{60,90}})));
+  Modelica.Thermal.HeatTransfer.Sources.PrescribedHeatFlow heaSur
+    annotation (Placement(transformation(extent={{80,-10},{60,10}})));
+  Modelica.Thermal.HeatTransfer.Sources.PrescribedHeatFlow heaMas
+    annotation (Placement(transformation(extent={{80,-90},{60,-70}})));
+  IBPSA.BoundaryConditions.WeatherData.Bus weaBus annotation (Placement(
+        transformation(extent={{-140,100},{-100,140}}),
+                                                      iconTransformation(extent={{-126,78},
+            {-62,142}})));
+  Modelica.Thermal.HeatTransfer.Sources.PrescribedTemperature TExt
+    "External air temperature"
+    annotation (Placement(transformation(extent={{-100,-10},{-80,10}})));
+
+  Modelica.Blocks.Math.Add solGai "Total solar heat gains"
+    annotation (Placement(transformation(extent={{-60,-80},{-40,-60}})));
+
+
+
+  BaseClasses.GlazedElements glaEle(
+    AWin=AWin,
+    winFra=winFra,
+    surTil=surTil,
+    surAzi=surAzi,
+    gFac=gFac) "Solar heat gains of glazed elements"
+    annotation (Placement(transformation(extent={{-100,-60},{-80,-40}})));
+  BaseClasses.OpaqueElements opaEle(
+    AWal=AWal,
+    UWal=UWal,
+    ARoo=ARoo,
+    URoo=URoo,
+    surTil={1.5707963267949,1.5707963267949,1.5707963267949,1.5707963267949,0},
+    surAzi={surAzi[1],surAzi[2],surAzi[3],
+        surAzi[4],0}) "Solar heat gains of opaque elements"
+    annotation (Placement(transformation(extent={{-100,-100},{-80,-80}})));
+  Modelica.Blocks.Interfaces.RealInput intGai "Internal heat gains"
+    annotation (Placement(transformation(extent={{-180,-140},{-140,-100}})));
```

Comment:
List all parameters, then all inputs, all output, then the instances, followed by the same order for any protected instances.

Reply:
Done